pkgs/development/tools/build-managers/bazel/default.nix

```diff
@@ -25,6 +32,5 @@ stdenv.mkDerivation rec {
     description = "Build tool that builds code quickly and reliably";
     license = stdenv.lib.licenses.asl20;
     maintainers = [ stdenv.lib.maintainers.philandstuff ];
-    platforms = [ "x86_64-linux" ];
```

Comment:
Same issue with clangStdenv in place of stdenv, alas.  Oh yeah, I also had to replace jdk with jdk8 explicitly (for some reason Darwin still has jdk7 as the default jdk)